### PR TITLE
Complete eu translation

### PR DESCRIPTION
**Pull request by erral**

### CODE REVIEW

File: locales/eu/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -355,17 +355,17 @@ msgstr "Ordezko helbidearen bidea → helburu helbidearen bidea (sorrera data et
 #: components/manage/Rules/Rules
 # defaultMessage: Applied to subfolders
 msgid "Applied to subfolders"
-msgstr ""
+msgstr "Azpikarpetei ondo aplikatu da"
```
Comment (ionlizarazu) [inline locales/eu/LC_MESSAGES/volto.po:358]: "Azpikarpetei ondo aplikatu zaie" ?

**Review by ionlizarazu — APPROVED** [review]

**Comment by sneridagh** [comment]
@erral please the flow always master->16.x.x !

**Comment by erral** [comment]
@sneridagh but for translations this does not work.

Now in the beginning it may work, but when there are things in master that are not merged to 16.x.x, the translations PRs can't be cherry-picked.

I think that the translations should be kept separate from the rule of development flow master -> 16.x.x

We had a small discussion the other day about this in slack I think.

**Comment by sneridagh** [comment]
@erral thanks for being on the top of this!

**Comment by erral** [comment]
I will add it to the docs

**Pull request merged**